bundles/org.openhab.binding.guntamatic/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -71,10 +71,73 @@
 		</properties>
 		<config-description-ref uri="thing-type:guntamatic:heatingsystem"/>
 	</thing-type>
+	<thing-type id="biosmart">
```

Comment:
Not a request of change but a big encouragement : your file is very big with a lot of thing types. The good practice in this case is to split to have one XML file for each thing type and another file with all the channel types.